plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerDelegate.java

```diff
@@ -163,13 +163,14 @@ private static IStatus hasAppEngineStandardFacet(IModule module) {
       if (AppEngineStandardFacet.hasAppEngineFacet(ProjectFacetsManager.create(module.getProject()))) {
         return Status.OK_STATUS;
       } else {
-        String errorMessage = NLS.bind(Messages.GAE_STANDARD_FACET_MISSING, module.getName(),
+        String errorMessage =
+            NLS.bind(Messages.getString("GAE_STANDARD_FACET_MISSING"), module.getName(),
```

Comment:
do we actually need NLS.bind here?
also, I think you'll want a NON-NLS-1 comment here